src/ewatercycle/analysis/__init__.py

```diff
@@ -46,38 +48,24 @@ def hydrograph(
     Optionally, the corresponding precipitation data can be plotted for
     comparison.
 
-    Parameters
-    ----------
-    discharge : pd.DataFrame
-        Dataframe containing time series of discharge data to be plotted.
-    reference : str
-        Name of the reference data, must correspond to a column in the discharge
-        dataframe. Metrics are calculated between the reference column and each
-        of the other columns.
-    precipitation : pd.DataFrame, optional
-        Optional dataframe containing time series of precipitation data to be
-        plotted from the top of the hydrograph.
-    dpi : int, optional
-        DPI for the plot.
-    title : str, optional
-        Title of the hydrograph.
-    discharge_units : str, optional
-        Units for the discharge data.
-    precipitation_units : str, optional
-        Units for the precipitation data.
-    figsize : (float, float), optional
-        With, height of the plot in inches.
-    filename : str or Path, optional
-        If specified, a copy of the plot will be saved to this path.
-    nbars : Int, optional
-        Number of bars to use for downsampling precipitation.
-    **kwargs:
-        Options to pass to the matplotlib plotting function
+    Args:
```

Comment:
Is it okay to leave this in google style? Or should it be reverted for consistency?

Reply:
I checked all docstrings and they are all in google style now.